src/theme/variables/airbitz.js

```diff
@@ -113,7 +114,7 @@ export const THEME = {
     CRYPTO_EXCHANGE_WALLET_LIST_ROW_OPAQUE_BLUE: 'rgba(14, 75, 117, 0.5)',
     FIO_ADDRESS_LIST_BORDER_BOTTOM: '#D8E2ED',
     FIO_ADDRESS_LIST_FONT: '#4B5158',
-    QR_CODE_THEME_INDEPENDENT_WHITE: 'white', // As the comment say, this should be theme independent
```

Comment:
If we have a `QR_CODE_BACKGROUND`, we need a `QR_CODE_FOREGROUND`. Please see the `<QrCode />` component which accepts these two colors as props.